great_expectations/core/expectation_configuration.py

```diff
@@ -1070,8 +1071,12 @@ def kwargs(self, value: dict) -> None:
         self._kwargs = value
 
     @property
-    def include_rendered_content(self) -> str:
-        return self._include_rendered_content
+    def rendered_content(self) -> dict:
+        return self._rendered_content
+
+    @rendered_content.setter
+    def rendered_content(self, value: dict) -> None:
+        self._rendered_content = value
```

Comment:
`rendered_content` is now a required property, but the key is dropped from serialization if it is `None`.